### ISSUE TITLE
Add a form to collect labels for type of bug

### ISSUE THREAD

**Issue by marco-c** [marco-c/bugzilla-data-collector#3]
Needed for https://github.com/marco-c/bugbug/issues/20.

**Comment by rhcu** [comment]
Just to clarify before starting: I think it should be similar to #7 but with more choices (Security, Crash, Hang, Incorrect rendering, etc [from https://github.com/mozilla/bugbug/issues/20]). Am I right?

**Comment by marco-c** [comment]
Correct! If possible, we should try to make the code more generic, so we don't have to manually add a new form every time.

For example, you could define a new file (experiments.js) where you store something like:
```
const EXPERIMENTS = [
{"regression_bug_nobug": { "name": "Regression/Bug/No-Bug", "choices": ["Regression", "Bug but not regression", ...],
{"qaneeded": {"name": "QA Needed", "choices": [...]}
...
];
```

Then, in the file where we add the forms you do something like:
```
for (let experiment of EXPERIMENTS) {
  // create form

  for (let choice of choices) {
    // create option
    // add option to form
  }
}
```

**Comment by rhcu** [comment]
Probably, I can create a new issue to make the form creation generic to make PR self-contained?

**Comment by marco-c** [comment]
Yes, good idea!
